datamodel/initdb.d/03_dcsa_im_v3_0.sql

```diff
@@ -263,7 +263,9 @@ CREATE TABLE dcsa_im_v3_0.booking (
     contract_quotation_reference varchar(35) NULL,
     incoterms varchar(3) NULL REFERENCES dcsa_im_v3_0.incoterms(incoterms_code),
     invoice_payable_at varchar(100) NULL REFERENCES dcsa_im_v3_0.location(id),
-    expected_departure_date timestamp NULL,
+    expected_departure_date date NULL,
+    expected_arrival_date_start date NULL CHECK ((expected_arrival_date_start IS NULL AND expected_arrival_date_end IS NULL) OR (expected_arrival_date_start IS NOT NULL) OR (expected_arrival_date_end IS NOT NULL) OR expected_arrival_date_start <= expected_arrival_date_end),
```

Comment:
I think this check will always return true and the start <= end will never be validated

Reply:
The check should probably be
(expected_arrival_date_start IS NULL) OR (expected_arrival_date_end IS NULL) OR expected_arrival_date_start <= expected_arrival_date_end)

Reply:
Are you sure - what happens when both start + end are Null - won't that give an exception when doing the `expected_arrival_date_start <= expected_arrival_date_end`?

Reply:
You are right - I will change the above